docs/preservation-package-spec.md

```diff
@@ -0,0 +1,34 @@
+# Preservation Package Specification
+
+All theses submitted to the ETD system will be sent to preservation submission storage after they have been published to DSpace. From preservation submission storage they will be ingested into ArchivesSpace via Archivematica by digital preservation/archives staff. Archivematica ingest requires a specific [submission structure](https://www.archivematica.org/en/docs/archivematica-1.13/user-manual/transfer/bags/#bags) and [metadata format](https://www.archivematica.org/en/docs/archivematica-1.13/user-manual/transfer/import-metadata/#metadata-bags) outlined below. Every thesis submitted to preservation must be packaged according to this specification.
+
+## Package structure
+
+Each thesis must be submitted as a single zipped bag according to the [bagit specification](https://datatracker.ietf.org/doc/html/rfc8493). The bag should include all files and metadata associated with the thesis, and must be structured as follows. See file [1721.1_123456-thesis.zip](1721.1_123456-thesis.zip) in this project's docs folder for an example bag structured according to these requirements.
+
+- _dspace_handle_-thesis.zip The top-level folder must be a zipped bag named with the
+  thesis's DSpace handle followed by "-thesis.zip"
+  - data/ There must be a folder in the bag named "data", containing all files associated with the thesis
+    - metadata/ There must be a folder in the "data" folder named "metadata", containing exactly one file of the metadata for the thesis
+      - metadata.csv The metadata file must be named "metadata.csv". See [Metadata File section](#metadata-file) below for details on the contents of this file
+    - _last-kerberos-DEG-DEP-YYYY_-thesis.pdf The data folder must at minimum contain the PDF representation of the thesis
+    - _last-kerberos-DEG-DEP-YYYY-supplemental1.txt_ The data folder may additionally contain any other files submitted with the thesis including supplementary files, signature pages, etc
+  - bag-info.txt Per the bagit specfication, the bag must contain a bag-info.txt file with the required contents
+  - bagit.txt Per the bagit specfication, the bag must contain a bagit.txt file with the required contents
+  - manifest-_algorithm_.txt Per the bagit specfication, the bag must contain at least one manifest-_algorithm_.txt file containing the checksum and file path of each file in the data folder, where _algorithm_ in the file name should be the cryptographic hash algorithm use to create the checksums, e.g. "sha256"
+  - tagmanifest-_algorithm_.txt Per the bagit specfication, the bag must contain at least one tagmanifest-_algorithm_.txt file containing the checksum and file path of each file in the top-level bag folder, where _algorithm_ in the file name should be the cryptographic hash algorithm use to create the checksums, e.g. "sha256"
```

Comment:
What? After reading this multiple times, I think it means we tagmanfest will contain hashes for `bag-info.txt`, `bagit.txt`, and `manifest-_algorithm_.txt`? Part of the confusion is that `tagmanifest-_algorithm_.txt` is also in the top level and can't contain it's own hash as adding that hash would change the hash...right?

Reply:
Yeah I'm not sure how specific we need to be about those files as they just follow exactly the bagit spec. I don't think we need to reiterate that here? (And using bagit libraries, which we will do, creates those files for us.)

Reply:
Hmmm. I'm not confident there are well maintained bagit libraries for ruby. However, this is a separate conversation. I'm comfortable with leaving the details here or just doing a more general "follow the bagit spec".